emctl/cmd/client/command/meshinstall/operator/csr.go

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2017, MegaEase
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package operator
+
+import (
+	"context"
+	"fmt"
+
+	installbase "github.com/megaease/easemeshctl/cmd/client/command/meshinstall/base"
+
+	certv1beta1 "k8s.io/api/certificates/v1beta1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// deployCSR deploys a new CertificateSigningRequest with approval action.
+func deployCSR(ctx *installbase.StageContext, csrPem, keyPem []byte) (CertPem []byte, err error) {
+	csr := &certv1beta1.CertificateSigningRequest{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "CertificateSigningRequest",
```

Comment:
certificates.k8s.io/v1beta1 CertificateSigningRequest is deprecated in v1.19+, please upgrade to certificates.k8s.io/v1 CertificateSigningRequest

Reply:
Fixed